### ISSUE TITLE
iOS 10 iphone 15 pro max

### ISSUE THREAD

**Issue by ndogtester** [leptos-null/MonthDay#1]
App Store says I purchased, but app is not on the Apple Watch. Can’t redownload and is not visible in my apps on the watch.

**Comment by leptos-null** [comment]
Sorry, I didn’t see this report earlier. Were you able to resolve this? Looks like #2 is a similar issue.

**Comment by ndogtester** [comment]
No, I had to go with a different app that worked! I would gladly switch back if it were working!

**Comment by leptos-null** [comment]
If you’re interested in trying out this solution, the user that opened #2 was able to resolve by re-downloading the app on the watch itself instead of through the iPhone

https://github.com/leptos-null/MonthDay/issues/2#issuecomment-2103025303
